s-port_flashing.md

```diff
@@ -1,2 +1,108 @@
-# S-PORT flashing
-TO-DO
+# S.Port Firmware Flashing
+
+## File Preparation
+
+- Download the firmware
+
+  ![](images/sport-flash-001-download-firmware.png)
+  
+- Place the firmware on the SDCARD in the FIRMWARE folder
+  
+  ![](images/sport-flash-104-sd-hc-firmware.bmp)
```

Comment:
How do I do this? Update the file in my fork and do a new pull request? Not sure how this works...